build/Band_Representations.jl

```diff
@@ -0,0 +1,260 @@
+using LinearAlgebra
+using Crystalline
+using StaticArrays
+import Crystalline:(get_lgirreps)
+
+function get_lgirreps(sgnum::Int64, D::Int64, timereversal::Bool)
+    if timereversal==false
```

Comment:
Silly, but this is stage 2 of the below 😄 ... (there's four such occurrences, as far I can see😄): 

![](https://i.redd.it/527m5aviag521.png)